android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -611,6 +890,33 @@ public void mediaStreamTrackSwitchCamera(String id) {
         });
     }
 
+    /**
+     * This serializes the transceivers current direction and mid and returns them
+     * for update when an sdp negotiation/renegotiation happens 
+     */
+    private ReadableArray getTransceiversSdpInfo(int id) {
```

Comment:
Let's be a bit more generic and call it just info? We can update a number of things.

Reply:
Done.